### DISCUSSION TITLE
Andrea Farias

### DISCUSSION THREAD

**Discussion by andrea-is-a** [life-itself/community#1049] [Introductions]
I'm a digital researcher, strategist and designer working at the meeting point between digital technologies, social justice and ecosystem regeneration. I have 8+ years of experience in the digital space, and have led projects for organizations like [Change.org](http://change.org/) and Mozilla. Previously, I founded [Pivotally](https://pivotally.co/), a product discovery agency helping software startups understand their customers and conduct product experiments.

About a year ago I started publishing my digital garden [Diome](https://diome.xyz/), which is a container to investigate the role of collaboration technologies in supporting the transition towards a regenerative civilization. Recently launched @thirdattractor (Instagram & Twitter), a project aiming to communicate Metacrisis concepts in an accessible format. Currently my journey is focused on learning, sharing & convening around creating a Regenerative Civilization.

**Comment by rufuspollock** [comment]
Welcome Andrea!

> Recently launched @thirdattractor (Instagram & Twitter), a project aiming to communicate Metacrisis concepts in an accessible format

Really resonate with that and big part of our second renaissance framing https://lifeitself.org/second-renaissance

Look forward to talking more.
